libkbfs/folder_branch_ops.go

```diff
@@ -564,8 +575,21 @@ func (fbo *folderBranchOps) getJournalPredecessorRevision(ctx context.Context) (
 	return jStatus.RevisionStart - 1, nil
 }
 
+// validateHead should be called with headLock taken.
+func (fbo *folderBranchOps) validateHead(
```

Comment:
the convention is to name this `validateHeadLocked`, have it take an `lState` variable, and call `fbo.headLock.AssertLocked(lState)`